lib/Data/Validate/Sanctions.pm

```diff
@@ -167,6 +181,15 @@ return 1 for yes, 0 for no.
 
 it will remove all non-alpha chars and compare with the list we have.
 
+=head2 is_sanctioned_hash
```

Comment:
In this case I also need to move all other pieces of documentation accordingly, so as you'd say